build.gradle

```diff
@@ -332,7 +332,7 @@ dependencies {
 
     runtimeOnly 'net.minidev:accessors-smart:2.4.7'
 
-    runtimeOnly 'org.apache.cxf:cxf-core:3.4.5'
+    runtimeOnly 'org.apache.cxf:cxf-core:3.5.5'
```

Comment:
we should update all cxf related dependencies here

Reply:
Good catch--fixed.